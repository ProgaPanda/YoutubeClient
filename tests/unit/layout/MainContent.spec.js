import { shallowMount } from '@vue/test-utils';
import MainContentLayout from '@/layout/MainContent.layout.vue';

describe('MainContentLayout', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(MainContentLayout, {});

    expect(wrapper.element).toMatchSnapshot();
  });
});
