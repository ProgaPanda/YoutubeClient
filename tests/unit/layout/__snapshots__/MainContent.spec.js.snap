// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContentLayout renders correctly 1`] = `
<main
  class="main-layout"
/>
`;
